rdrf/registry/patients/models.py

```diff
@@ -84,11 +84,15 @@ def __unicode__(self):
 class PatientManager(models.Manager):
 
     def get_by_registry(self, registry):
-        return self.model.objects.filter(rdrf_registry__in=registry)
+        registries = registry if hasattr(registry, '__iter__') else (registry,)
```

Comment:
I don't like such overloading. And the list argument form doesn't seem to be used anyway. What about `def get_by_registry(self, *registries):` ?